src/oom/oomctl.c

```diff
@@ -55,15 +56,14 @@ static int dump_state(int argc, char *argv[], void *userdata) {
 
         pager_open(arg_pager_flags);
 
-        r = sd_bus_call_method(
+        r = bus_call_method(
                         bus,
-                        "org.freedesktop.oom1",
-                        "/org/freedesktop/oom1",
-                        "org.freedesktop.oom1.Manager",
+                        bus_systemd_mgr,
```

Comment:
This is wrong. You're calling pid1's `DumpByFileDescriptor` method instead of `oomd`'s.